lib/mumukit/platform/application.rb

```diff
@@ -47,6 +47,10 @@ class Basic < Mumukit::Platform::Application
     def organic_uri(_organization)
       uri
     end
+
```

Comment:
I don't think that Basic apps need to respond to that method, but I added it to keep it consistent with organic_uri decision